build/create-env.ps1

```diff
@@ -4,30 +4,43 @@
 <#
     .SYNOPSIS
         Create Conda environment(s) for given package directories
+        Optionally, use CondaEnvironmentSuffix to specify a special environment file with name environment<CondaEnvironmentSuffix>.yml.
 #>
 
 param(
-  [string[]] $PackageDirs
+  [string] $PackageName,
+  [string] $CondaEnvironmentSuffix
 )
 
-if ($null -eq $PackageDirs) {
+if ('' -eq $PackageName) {
+  # If no package dir is specified, find all packages that contain an environment.yml file
   $parentPath = Split-Path -parent $PSScriptRoot
-  $PackageDirs = Get-ChildItem -Path $parentPath -Recurse -Filter "environment.yml" | Select-Object -ExpandProperty Directory | Split-Path -Leaf
-  Write-Host "##[info]No PackageDir. Setting to default '$PackageDirs'"
+  $PackageNames = Get-ChildItem -Path $parentPath -Recurse -Filter "environment.yml" | Select-Object -ExpandProperty Directory | Split-Path -Leaf
```

Comment:
should include CondaEnvironmentSuffix if provided.

Reply:
This would cause duplicates. I don't think we will ever have a package without the default "environment.yml" file.